runtime/polkadot/src/lib.rs

```diff
@@ -1760,6 +1767,10 @@ sp_api::impl_runtime_apis! {
 			})
 		}
 
+		fn session_ee_params_by_parent_hash(parent_hash: Hash) -> Option<ExecutorParams> {
```

Comment:
I assume you don't this in polkadot runtime initially. If you tag the method with v3 you won't need to provide implementation here.